packages/farcaster-js-neynar/src/v2/openapi/spec.yaml

```diff
@@ -270,7 +286,6 @@ components:
       type: object
       required:
         - signer_uuid
-        - text
```

Comment:
Huh. `text` is now optional when posting a cast?

Reply:
🤷‍♂️